src/inc/clrconfigvalues.h

```diff
@@ -836,6 +836,7 @@ RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_ProfAPI_ValidateNGENInstrumentation, W("Pro
 
 #ifdef FEATURE_PERFMAP
 RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_PerfMapEnabled, W("PerfMapEnabled"), 0, "This flag is used on Linux to enable writing /tmp/perf-$pid.map. It is disabled by default", CLRConfig::REGUTIL_default)
+RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_PerfMapIgnoreSignal, W("PerfMapIgnoreSignal"), 0, "When perf map is enabled, this option will configure the specified signal to be accepted and ignored.  This allows it to be used as a marker in the perf logs.  It is disabled by default", CLRConfig::REGUTIL_default)
```

Comment:
Hmm..  
I can leave it dependent on FEATURE_PERFMAP.  I can enable it independent to the PerfMapEnabled, if t is is generally useful.  

Perf has unique requirement on how the signal should be ignored.  Normally, you would tell the kernel to ignore the signal by not delivering it.  I had assumed this meant the config should be unique to perf, only needed to handle one signal, and was not generally useful..

Most common orthogonal use cases would be prefer the kernel ignoring a signal.  This is significantly faster for the kernel as it doesn't need to do anything.  

@adityamandaleeka @janvorli What do you think?